js/client-battle.js

```diff
@@ -555,6 +562,10 @@
 			if (type === 'movetarget') {
 				requestTitle += 'At who? ';
 
+				if (this.request && this.request.side) {
+					pos += Math.floor((parseInt(this.side.charAt(1), 10) - 1) / 2);
+				}
```

Comment:
?

Reply:
It looks like this is doing the same thing as `activePos` but different?

Reply:
Yeah. They both do the same thing in a normal Multi Battle, but whereas this supports a theoretical Multi-Triple Battle with a p5 and p6, `activePos` is written to support a theoretical 2v2v2v2 kind of battle. I couldn't tell you why, though.

Reply:
I don't really care which one you go with, but probably they should match for readability reasons.